src/components/MessageLivestream.js

```diff
@@ -3,10 +3,27 @@ import { withChannelizeContext } from '../context';
 import { dateSeparatorParser } from '../utils'
 import { Avatar } from "./Avatar";
 import { LANGUAGE_PHRASES } from "../constants";
+import { OutsideClickHandler } from './OutsideClickHandler';
 
 class MessageLivestream extends Component {
 	constructor(props) {
   	super(props);
+  	this.state = {
+  		showMoreOptions: false
+		}
+	}
+
+	toggleMoreOptions = () => {
+    this.setState((state) => ({
+    		showMoreOptions: !state.showMoreOptions
+    }));
```

Comment:
@pardeep-kumar-bs why `toggleMoreOptions` function in constructor, you should put this outside of constructor.

Reply:
No. It is not.